src/resolvers/resolver.cr

```diff
@@ -44,7 +44,7 @@ module Shards
       end
     end
 
-    protected def install_path
+    def install_path
```

Comment:
It's not "public" because shards is an application not a library. It doesn't have to guarantee anything except it's command-line behaviour.